tests/migrations/test_migrations_squash_noop/0001_initial.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
Why do you need operations in here? Shouldn't a single RunPython operation in `0002` be enough to squash `0001` and `0002`?